server/app/src/main/java/com/ams/app/config/ASecurityConfig.java

```diff
@@ -0,0 +1,107 @@
+package com.ams.app.config;
+import com.ams.app.security.JwtAuthenticationFilterdemo;
+import com.ams.app.service.MyUserDetailsService;
+import com.ams.app.util.JwtUtil;
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Lazy;
+import org.springframework.http.HttpMethod;
+import org.springframework.security.authentication.AuthenticationManager;
+import org.springframework.security.authentication.AuthenticationProvider;
+import org.springframework.security.authentication.dao.DaoAuthenticationProvider;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
+import org.springframework.security.config.http.SessionCreationPolicy;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+import org.springframework.security.web.AuthenticationEntryPoint;
+import org.springframework.security.web.authentication.UsernamePasswordAuthenticationFilter;
+
+@Configuration
+@EnableWebSecurity
+@RequiredArgsConstructor(onConstructor = @__({@Autowired, @Lazy}))
+public class ASecurityConfig extends WebSecurityConfigurerAdapter {
```

Comment:
What is meant by ASecurityConfig? Rename it with a meaningful name

Reply:
While writing different configs, mistakenly named. Renamed now to SecurityConfig